includes/core/classes/class-assets.php

```diff
@@ -253,6 +241,18 @@ public function editor_enqueue_scripts(): void {
 		);
 
 		wp_set_script_translations( 'gatherpress-editor', 'gatherpress', GATHERPRESS_CORE_PATH . '/languages' );
+
+		$asset = $this->get_asset_data( 'admin' );
+
+		wp_enqueue_script(
+			'gatherpress-admin',
+			$this->build . 'admin.js',
+			$asset['dependencies'],
+			$asset['version'],
+			true
+		);
```

Comment:
Might be better to just combine `gatherpress-admin` with `gatherpress-editor`. I think the separation of these files was an oversight early in the project.

Reply:
Ok. I combined the files into one, without doing anything further, because _I know_ we will have to revisit this code for the next release, when we look into https://github.com/GatherPress/gatherpress/issues/626